modules/simple_forms_api/app/services/simple_forms_api/form_remediation/s3_client.rb

```diff
@@ -117,15 +113,20 @@ def generate_presigned_url(type: upload_type)
       end
 
       def s3_upload_file_path(type)
+        extension = File.extname(archive_path)
```

Comment:
What changed to make this appropriate? Or is it just an oversight from previously?

Reply:
Just an oversight from a previous change.